VikingQuiz.UI/src/Components/LoginFormComponent/LoginFormComponent.tsx

```diff
@@ -13,7 +13,7 @@ class LoginFormComponent extends FormComponent {
       <div className="form-body container-fluid">
         {this.state.inputs.map((input: InputData) => this.renderInput(input))}
         <div className="forget-remember col-xs-12">
-          <a href="#" className="forget-pass col-xs-6">FORGOT PASSWORD</a>
+          <a href="/forgot" className="forget-pass col-xs-6">FORGOT PASSWORD</a>
```

Comment:
Please extract the string into a constant so we can change it easily in the future if it is necessary.